install/kubernetes/helm/istio/charts/ingress/templates/service.yaml

```diff
@@ -23,6 +23,7 @@ spec:
   type: {{ .Values.service.type }}
   selector:
     istio: ingress
+    app: {{ template "ingress.name" . }}
```

Comment:
you missed the gateways

Reply:
but gateways service has not label app
https://github.com/istio/istio/blob/9789a6617e75b530980de62a61f1f28202896d6d/install/kubernetes/helm/istio/charts/gateways/templates/service.yaml#L13-L19